be/src/storage/union_iterator.cpp

```diff
@@ -61,6 +61,7 @@ class UnionIterator final : public ChunkIterator {
     Status do_get_next(Chunk* chunk) override;
     Status do_get_next(Chunk* chunk, std::vector<uint32_t>* rowid) override;
     Status do_get_next(Chunk* chunk, std::vector<uint64_t>* rssid_rowids) override;
+    Status do_get_next(Chunk* chunk, std::vector<RowSourceMask>* source_masks) override { return do_get_next(chunk); }
```

Comment:
No need to consider `source_masks` ?

Reply:
> No need to consider `source_masks` ?

I think it's unnecessary. The usage of `union_iterator` is to read data in the order of segments and it is no need to record the read order.

Reply:
add comments in `rowset_merger`?

Reply:
> add comments in `rowset_merger`?

done.